app/kubemci/pkg/gcp/forwardingrule/forwardingrulesyncer_test.go

```diff
@@ -580,8 +580,9 @@ func TestRemoveClustersFromStatus(t *testing.T) {
 			t.Errorf("%s", err)
 		}
 		// Update status to remove one cluster.
-		if err := frs.RemoveClustersFromStatus([]string{"cluster1"}); err != nil {
-			t.Errorf("unexpected error in updating status to remove clusters: %s", err)
+		err := frs.RemoveClustersFromStatus([]string{"cluster1"})
+		if c.shouldErr != (err != nil) {
+			t.Errorf("unexpected error in updating status to remove clusters, expected err != nil: %v, actual err: %s", c.shouldErr, err)
```

Comment:
In that case this will read as "unexpected error in updating status to remove clusters, expected err != nil: true, actual err: nil". Seems fine? I dont want to put 2 special error messages.

Reply:
Hmm. I think the test output and this code would be more readable if you had two branches. If this test failed and I had to look at it without having seen it before, it would take me some time to parse out all of the boolean conditions to actually understand the failure, which is not ideal in a test.

If you wanted to leave one branch, I think the failure output would be clearer if you replaced `expected err != nil: %v` with `expected err %s nil:` and a branch that set either `==` or `!=` as the replacement for `%s`.

Reply:
Changed to `"expected err != nil: %v, actual err != nil: %v, err: %s", c.shouldErr, err != nil, err`